### PR TITLE
Update Seating Chart for Canvas Usage (Solves #6) 

### PR DESCRIPTION
**Pull request by joshuazweig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,10 @@ For sample inputs, see the `out/demo/` directory.
...
 
 *   Photo roster
-    -   Go to Roster > Print as Single Column. Cancel the print dialog.
+    -   Go to Photo Roster in Canvas
     -   Use File > Save Page As. In the Format drop-down, select "Web Page,
         complete".
-    -   Navigate to your working directory.
-    -   Name the file `roster_your-slug.html` and press Save. (This will also
-        create a directory to store the photos.)
+    -   Ensure there is a directory named `photoroster.roster_files/` containing the student's images in the format `<uni>.jpg`
```
Comment (kevin1) [inline README.md]: can we make this follow the naming convention?

File: seatingchart.py [hunk 2]
```diff
@@ -68,15 +68,18 @@ def working_dir_path(name, slug, extension):
 ASSIGN_LAST  = assign_last_path  if os.path.isfile(assign_last_path)  else "/dev/null"
 
 # 3) a HTML page matching the courseworks print as single column roster feature, with student pictures
-roster_path = working_dir_path("roster", args.slug, "html")
-assert_file_exists(roster_path)
-ROSTER_PAGE = roster_path
+# roster_path = working_dir_path("roster", args.slug, "html")
+# assert_file_exists(roster_path)
+# ROSTER_PAGE = roster_path
```
Comment (kevin1) [inline seatingchart.py]: we can just delete these lines and get it back from the git history if it's actually needed

File: seatingchart.py [hunk 3]
```diff
@@ -102,7 +106,13 @@ def working_dir_path(name, slug, extension):
     assign_first = [x.strip() for x in open(ASSIGN_FIRST).readlines()]
 
 
-    students = [tuple(s) for s in csv.reader(open(STUDENT_LIST))][1:]
+    students = [tuple(s) for s in csv.reader(open(STUDENT_LIST, "rU"))][1:]
+
+    # Assign students photos 
+    for student in students:
+        uni = student[0]
+        photos[uni] = PHOTO_DIR + uni + ".jpg"
```
Comment (kevin1) [inline seatingchart.py]: let's use os.path.join in case someone wants to run this program on windows

File: seatingchart.py [hunk 4]
```diff
@@ -166,19 +176,8 @@ def working_dir_path(name, slug, extension):
...
     output.writerow(list(uni)[:2] + [seat])
 
-
-#Load picture files
-soup = BeautifulSoup(open(ROSTER_PAGE), "html.parser")
-photos = {}
-for img in soup.findAll(class_="rosterImage"):
-    url = img["src"]
-    uni = img.findParent().findNextSibling().findNextSibling().string
-    photos[uni] = url
```
Comment (kevin1) [inline seatingchart.py]: was this the only use of beautiful soup? we can get rid of the import so people don't need to install that library anymore

File: README.md [hunk 5]
```diff
@@ -33,12 +33,10 @@ For sample inputs, see the `out/demo/` directory.
     -   Rename it `roster_your-slug.csv`.
```
Comment (kevin1) [inline README.md]: Do we still use old cw for the text roster?

**Comment by hmontero1205** [comment]
outdated

**Pull request closed without merging**